pyanaconda/modules/network/nm_client.py

```diff
@@ -950,6 +950,39 @@ def finish_callback(nm_client, result, sync_queue):
     return ret
 
 
+def clone_connection_sync(nm_client, connection, con_id=None, uuid=None):
```

Comment:
I'm missing docstring here. It would be great to have an explanation here.

Reply:
Also I think this should be handled by a decorator somehow. If there is no available we should create it.

Reply:
I've added the docstring porting commit 010e5fc8dc6de9fd7ecd20d4bcd0815dfa8a1555 .
The decorator would be something for upstream, but we will to have to rework this part anyway.